doors_dashboards/assets/help.md

```diff
@@ -0,0 +1,149 @@
+for browsing and analyzing varied datasets using interactive visualizations. 
+The datasets that can be visualized with the Dashboards App are vector data, such as 
+in-situ measurements.
+Any such dataset within the Dashboards App is called a collection. 
+
+The Dashboards Application is a collection of several dashboards.
+Each dashboard consists of different elements - so far, each one at least of a map - 
+and may handle different collections.
+Other elements are typically plots that may help to gain further insight into a
+collection.
+The overview button in the top left brings up the available dashboards.
+
+<!-- <img src="doors_dashboards/assets/_static/screenshot_dashboard_selection_pnl.png" 
+alt="Dashboard" style="width:60em;display:block;margin-left:auto;
+margin-right:auto"/> -->
+![Dashboard](_static/screenshot_dashboard_selection_pnl.png)
+
+Functionality
+=============
+
+The Dashboards App provides access to a range of functions such as weather forecasts, 
+time series, scatter plots, point and line graphs, and trajectory displays. 
+It allows for fast load times and smooth interactions, benefiting the overall experience 
+for users.
+
+The Base Map
+------------
+
+The Map is the one element that you may find in every dashboard.
+It will either show locations from elements of one or more collections, 
+or other specifically denoted locations.
+To zoom, either use the buttons in the top right bar of the map (becomes visible on 
+hovering) or your mouse wheel. 
+
+<!-- <img src="doors_dashboards/assets/_static/screenshot_map_window_features.png" 
+alt="Bulgaria Dashboard" style="width:60em;display:block;margin-left:auto;
+margin-right:auto"/> -->
+![Bulgaria Dashboard](_static/screenshot_map_window_features.png)
+
+A dashboard visualizes data from the datasets on top of a basemap.
+For zooming hover the mouse on map and the buttons in the top right corner will show 
+up or the zooming function of your computer mouse.
+In the upper left corner a legend to the mapped collection is available. 
+To toggle the display of collections on the map, click a collection's name in the 
+upper left corner.
+Double-clicking will cause that only the selected collection is shown, another 
+double-click brings up the other collections again.
+
+<!-- <img src="doors_dashboards/assets/_static/screenshot_map_legend.png" 
+alt="Legend Dashboard" style="width:60em;display:block;margin-left:auto;
+margin-right:auto"/> -->
+![Legend Dashboard](_static/screenshot_map_legend.png)
+
+To change the map theme click on the `Settings` button on the top right of the app. 
+`Carto Positron` is the default map theme applied.
+
+<!-- <img src="doors_dashboards/assets/_static/screenshot_map_setting.png" 
+alt="Settings Dashboard" style="width:60em;display:block;margin-left:auto;
+margin-right:auto"/> -->
+![Settings Dashboard](_static/screenshot_map_setting.png)
+
+Plots
+-----
+
+When the dashboard offers also plots in addition to the map, you can switch between
```

Comment:
I think we can remove the word "When"

Reply:
I'd like to keep it. There are a few dashboards who only show a map, and those don't have the collections drop down menu,